src/main/resources/static/js/location.js

```diff
@@ -14,64 +16,64 @@ function createLocation() {
 		var myloc = {
 				name: name,
 		        description: desc,
-		        encoding_TYPE: 'application/vnd.geo+json',
+		        encoding_TYPE: "application/vnd.geo+json",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs. (no-mixed-spaces-and-tabs)](https://app.codacy.com/app/FraunhoferIOSB/ChillImport/pullRequest?prid=4696407)